README.rst

```diff
@@ -130,6 +136,16 @@ Array specifiers ( *idx* ):
 | ``[*]``                                 | any array index                                                                       |
 +-----------------------------------------+---------------------------------------------------------------------------------------+
 
+Array filter ( *filter* ):
+
++----------------------------------------------------+----------------------------------------------------------------------------------------+
+| Syntax                                             | Meaning                                                                                |
++====================================================+========================================================================================+
+| ``[``\ ``?``\ *jsonpath*\ ``operator``\ *n*\ ``]`` | Array with row where the jsonpath value *operator* than *n*' (may be &-seprated list), |
```

Comment:
It would be nice to have `[?`*expr*`]` with a definition of the allowed expressions separate.

A first step which would avoid having complex expressions would just be to replace *n* with *constant* that can be a constant of any type.